ui/apps/platform/src/hooks/useFetchDeploymentCount.ts

```diff
@@ -12,14 +13,14 @@ type UseFetchDeploymentCount = {
 };
 
 const DEPLOYMENT_COUNT_FOR_CLUSTER = gql`
-    query deployments($query: String) {
-        count: deploymentCount(query: $query)
+    query getDeploymentCountForCluster($id: ID!) {
+        cluster(id: $id) {
+            count: deploymentCount
+        }
```

Comment:
I can't tell why this file is changed in this PR ?

Reply:
The changes for this file don't really need to happen. I ended up refactoring a bit as I was trying to figure out the issue